.github/workflows/check-config.yml

```diff
@@ -0,0 +1,18 @@
+on: [ pull_request ]
+
+jobs:
+  check-config:
+    runs-on: ubuntu-latest
+    outputs:
+      targets: ${{ steps.set-matrix.outputs.targets }}
+    steps:
+      - uses: actions/checkout@v2
+      - name: check if config.json is sorted alphabetically
+        run: |
+          jq -r ".[].target" config.json > entries.txt
+          sort entries.txt > entries_sorted.txt
```

Comment:
perhaps use `sort -u`, which will also make this check complain if there are duplicate entries.

Reply:
Good point, I hadn't thought of duplicates.